frontend/src/lib/components/CopyToClipboard.tsx

```diff
@@ -77,34 +66,3 @@ export function CopyToClipboardInline({
         <>{content}</>
     )
 }
-
-export function CopyToClipboardInput({
```

Comment:
@Twixes Can you confirm if we still need this? 😮 It doesn't look like it's being used anywhere and seems like `CopyToClipboardInline` takes care of it?

Reply:
This is good, we don't use this input style anymore!